deps/wazuh_testing/wazuh_testing/end_to_end/vulnerability_detector.py

```diff
@@ -131,6 +131,34 @@ def get_alerts_by_agent(alerts, regex) -> Dict:
     return alerts_vuln_by_agent
 
 
+def parse_vulnerability_detector_alerts(alerts) -> Dict:
+    """
+    Get specific alerts by agent.
+
+    Args:
+        alerts (list): List of alerts.
+        regex (str): Regular expression to match the alerts.
+
+    Returns:
+        dict: Dictionary containing the alerts by agent.
+    """
+    vulnerability_detector_alerts = {}
+    vulnerability_detector_alerts['affected'] = []
+    vulnerability_detector_alerts['mitigated'] = []
+
+    regex_affected = 'CVE.*? affects.*"?'
+    rege_mitigated = "The .* that affected .* was solved due to a package removal"
```

Comment:
This is very general, could it be declared outside?

Reply:
Done e49f94aa404991848d82b074c67a8df884e9bfa0